src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -3889,6 +3827,6 @@ private static SocketError GetSocketErrorFromFaultedTask(Task t)
             };
         }
 
-        #endregion
+#endregion
```

Comment:
Maybe we should just delete all these regions markers.

Reply:
Done